src/lib/RoutingManager.ts

```diff
@@ -70,10 +70,8 @@ class RoutingManager {
     this.router.write(route);
   }
 
-  public dispose({ helper, state }): void {
-    if (this.router.dispose) {
```

Comment:
I've removed the guard since now the method is required. Note that he method might be missing since the router could be implemented in user-land. At the moment it's fine since we already use the others function without a check: `read`, `write`, `createURL`, ... We can add validation at some point on development mode.